docs/examples/Space_Robot.rst

```diff
@@ -0,0 +1,792 @@
+
+
+
+
+.. math::
+
+    \displaystyle \left[ x, \  y, \  z, \  q_{1}, \  q_{2}, \  q_{3}\right]
+
+
+
+.. code:: ipython3
+
+    sys.speeds
+
+
+
+
+.. math::
+
+    \displaystyle \left[ u_{x}, \  u_{y}, \  u_{z}, \  u_{1}, \  u_{2}, \  u_{3}\right]
+
+
+
+.. code:: ipython3
+
+    sys.states
+
+
+
+
+.. math::
+
+    \displaystyle \left[ x, \  y, \  z, \  q_{1}, \  q_{2}, \  q_{3}, \  u_{x}, \  u_{y}, \  u_{z}, \  u_{1}, \  u_{2}, \  u_{3}\right]
+
+
+
+.. code:: ipython3
+
+    sys.initial_conditions = {
+                              x: 0.0,
+                              y: 0.0,
+                              z: 0.0,
+                              q1: 0.0,
+                              q2: 0.0,
+                              q3: 0.0,
+                              ux: 0.2,
+                              uy: 0.0,
+                              uz: 0.0,
+                              u1: 0.0,
+                              u2: 0.0,
+                              u3: 0.5
+                             }
+
+.. code:: ipython3
+
+    sys.specifieds_symbols
+
+
+
+
+.. math::
+
+    \displaystyle \left\{\left|{F}\right|_{x}, \left|{F}\right|_{y}, \left|{F}\right|_{z}, \left|{T}\right|_{1}, \left|{T}\right|_{2}, \left|{T}\right|_{3}\right\}
+
+
+
+.. code:: ipython3
+
+    sys.specifieds = {
+                      Fx_mag: 0.0,
+                      Fy_mag: 0.0,
+                      Fz_mag: 0.0,
+                      T1_mag: 0.0,
+                      T2_mag: 0.0,
+                      T3_mag: 0.0
+                     }
+
+.. code:: ipython3
+
+    states = sys.integrate()
+
+.. code:: ipython3
+
+    import matplotlib as mpl
+    mpl.rcParams['figure.dpi'] = 200
+    
+    mpl.rc('font',**{'family':'serif','sans-serif':['Computer Modern Roman']})
+    ## for Palatino and other serif fonts use:
+    #rc('font',**{'family':'serif','serif':['Palatino']})
+    mpl.rc('text', usetex=True)
+    
+    from matplotlib.pyplot import cm
+    color=cm.rainbow(np.linspace(0,1,12))
+    
+    from cycler import cycler
+    mpl.rcParams['axes.prop_cycle'] = cycler(color=color)
+    
+    mpl.rcParams.update({'figure.autolayout': True})
+    
+    mpl.rcParams.update({'font.size': 12})
+    
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D, art3d
+
+.. code:: ipython3
+
+    fig, ax = plt.subplots()
+    ax.plot(sys.times, states)
+    ax.set_xlabel('{} [s]'.format(sm.latex(t, mode='inline')));
+    ax.set_ylabel('States');
+    ax.legend(['$x$', '$y$', '$z$', '$q_1$', '$q_2$', '$q_3$', '$u_x$', '$u_y$', '$u_z$', '$u_1$', '$u_2$', '$u_3$'], fontsize=10)
+    plt.show()
+
+
+
+.. image:: Space_Robot_files/Space_Robot_53_0.png
+
+
+3D Visualization
+----------------
+
+.. code:: ipython3
+
+    from pydy.viz.shapes import Cube, Cylinder, Sphere, Plane
+    from pydy.viz.visualization_frame import VisualizationFrame
+    from pydy.viz import Scene
+    from ipywidgets import Image, Video
+    import pythreejs as pjs
```

Comment:
The hack to use pythreejs is no longer needed. Please update to use the simpler form in the development version of pydy. The other documentation examples show how to do this:

https://pydy.readthedocs.io/en/latest/index.html#examples